app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I know this file shouldn't be on Git Hub, but I am abusing the fact that this is an academic project and that this is a private repository to make our lives easier.